src/plugins/gui/amountQuotePlugin.ts

```diff
@@ -34,10 +36,24 @@ type PaymentTypeProviderPriorityMap = ReturnType<typeof asPaymentTypeProviderPri
 
 type PriorityArray = Array<{ [pluginId: string]: boolean }>
 
+interface ConvertValueInternalResult {
+  stateManagerUpdate?: Partial<EnterAmountState>
+  bestError?: BestError
+  value?: string
+}
+type InternalFiatPluginEnterAmountParams = FiatPluginEnterAmountParams & {
+  convertValueInternal: (sourceFieldNum: number, value: string, stateManager: StateManager<EnterAmountState>) => Promise<ConvertValueInternalResult>
+}
+
 const providerFactories = [banxaProvider, bityProvider, kadoProvider, moonpayProvider, mtpelerinProvider, paybisProvider, simplexProvider]
 
 const DEFAULT_FIAT_AMOUNT = '500'
 
+// Amount that we will attempt to get a quote for if the user presses the MAX button
+// This is an arbitrary large number that should be larger than any reasonable quote
+// in either fiat or crypto
```

Comment:
/** Amount that we will attempt to get a quote for if the user presses the MAX button
This is an arbitrary large number that should be larger than any reasonable quote
in either fiat or crypto */
```suggestion
// Amount that we will attempt to get a quote for if the user presses the MAX button
// This is an arbitrary large number that should be larger than any reasonable quote
// in either fiat or crypto
```

Using this comment format instead will show the comment on hover